src/components/Navigation/SectionLink.jsx

```diff
@@ -40,10 +43,14 @@ class SectionLink extends React.Component {
     return className
   }
 
+  getActiveClassName () {
+    return this.hasErrors() ? '' : 'usa-current'
```

Comment:
Hmm, this is manipulating the classes to accommodate the CSS, no? I think we want it to have `usa-current` when a user is looking at that subsection, regardless of whether there are errors or not.

Reply:
Just want to make sure I'm following this. What you are describing above is how it currently functions, when a user goes to `Review`, `usa-current` is added which applies the blue line to the left of the link. Are you saying we still want `usa-current` added, but for it not to appear as the blue line?

Reply:
Correct!